plexapi/library.py

```diff
@@ -2849,6 +2861,137 @@ def _loadData(self, data):
+            raise BadRequest('Collection must be a Managed Recommendation to be removed')
+        if not self.deletable:
+            raise BadRequest('%s ManagedHub cannot be removed' % self.title)
+        key = '/hubs/sections/%s/manage/%s' % (self._librarySectionID, self.identifier)
+        self._server.query(key, method=self._server._session.delete)
+        self.identifier = None
+
+    def updateVisibility(self, recommended=None, home=None, shared=None):
+        """ Update the hub's visibility settings.
+
+            Parameters:
+                recommended (bool): True to make visible on your Library Recommended Page. False to Remove. Default None.
+                home (bool): True to make visible on your Home Page. False to Remove. Default None.
+                shared (bool): True to make visible on your Friends' Home Page. False to Remove. Default None.
+
+            Example:
+
+                .. code-block:: python
+
+                    hub.updateVisibility(recommended=True, home=True)
+        """
+        changed = False
+        if recommended is not None and ((recommended and not self.promotedToRecommended)
+                                        or (not recommended and self.promotedToRecommended)):
+            self.promotedToRecommended = True if recommended else False
+            changed = True
+        if home is not None and ((home and not self.promotedToOwnHome)
+                                 or (not home and self.promotedToOwnHome)):
+            self.promotedToOwnHome = True if home else False
+            changed = True
+        if shared is not None and ((shared and not self.promotedToSharedHome)
+                                   or (not shared and self.promotedToSharedHome)):
+            self.promotedToSharedHome = True if shared else False
+            changed = True
+        if changed:
+            args = {
+                "promotedToRecommended": 1 if self.promotedToRecommended else 0,
+                "promotedToOwnHome": 1 if self.promotedToOwnHome else 0,
+                "promotedToSharedHome": 1 if self.promotedToSharedHome else 0,
+            }
+            if self._collection_key:
+                args["metadataItemId"] = self._collection_key
+                key = '/hubs/sections/%s/manage%s' % (self._librarySectionID, utils.joinArgs(args))
+                self._server.query(key, method=self._server._session.post)
+                load_key = '/hubs/sections/%s/manage?metadataItemId=%s' % (self._librarySectionID, self._collection_key)
+                self._loadData(self._server.query(load_key)[0])
+            else:
+                key = '/hubs/sections/%s/manage/%s%s' % (self._librarySectionID, self.identifier, utils.joinArgs(args))
+                self._server.query(key, method=self._server._session.put)
+
+    def promoteRecommended(self):
+        """ Make the hub visible on your Library Recommended Page. """
+        self.updateVisibility(recommended=True)
```

Comment:
Add `return` to each method to allow method chaining.
```suggestion
        return self.updateVisibility(recommended=True)
```